api/opds2/publication.py

```diff
@@ -173,6 +176,14 @@ def parseEditionToPublication(self, editionRecord):
                     'type': link.media_type,
                     'rel': 'http://opds-spec.org/acquisition/open-access'
                 })
+                # Read Online Link
+                if 'reader' in link.flags:
+                    if link.flags['reader'] == True:
+                        self.addLink({
+                        'href': 'https://drb-qa.nypl.org/read/{}'.format(link.id),
```

Comment:
You commented out the `host` line above but we need to keep it (or the current host into this method call) because we shouldn't hard-code the host here (should be QA or production).

Reply:
I'll uncomment the host line. I commented it out in the first place due to wanting to test specifically when the url was 'drb-qa' and I was getting environment errors with the host variable originally.